airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/util/ConnectorExceptionUtil.java

```diff
@@ -88,22 +85,22 @@ public static <T extends Throwable> void logAllAndThrowFirst(final String initia
     }
   }
 
-  private static Predicate<Throwable> getConfigErrorPredicate() {
-    return e -> e instanceof ConfigErrorException;
+  private static boolean isConfigErrorException(Throwable e) {
+    return e instanceof ConfigErrorException;
```

Comment:
Thanks for this change and others like it. There's really no point in declaring static variables containing Function<> and friends over declaring a method.

Reply:
+1